### ISSUE TITLE
Zinio DRM Remover will not start

### ISSUE THREAD

**Issue by Lublio** [Metalnem/zinio#10]
I have tried to run zinio in Windows 10 Pro as Administrator with Antivirus (Kaspersky) disabled and the same again in safe mode with networking. In both cases the command prompt screen seems to open very momentarily (only a brief flash) on the screen but dies immediately. Can you help please?

**Comment by Metalnem** [comment]
This looks to me like you just double-clicked the executable. Did you actually follow the instructions in the README file?

**Comment by Lublio** [comment]
Dear Metalnem,
you are right I did initially double click zinio.exe in an attempt to run it. I am just an ordinary Windows PC user who rarely uses command line tools such as yours, so really needed more of a step by step set of instructions. Fortunately, following your prompt, a bit of Googling, some trial and error and the realization that I had forgotten to whitelist zinio.exe in Cryptoprevent finally reached a successful result. I am still a little puzzled by a couple of things though. I couldn't get Windows to "see" zinio.exe when I copied it to the system32 folder. I couldn't visually see it there either (I have hidden files and folders set to be visible). However Teracopy found it and had to overwrite it when I tried again to copy it to there. When I tried running it the string of characters you show in your readme section didn't appear. I copied it to the Main "Windows " folder in the end and ran it from there using CD .. to move up a driectory. Rightly or wrongly I used the "start" command first then found that I had to enter "zinio.exe" into command prompt followed by the email address and password commands as per your instrauctions. The result downloaded into a new folder within the Windows folder.

Finally, thank you for providing this tool. If a dunce such as myself can use it anyone can.

Regards, Lublio
